devops/all-tests-job-template.yml

```diff
@@ -44,7 +44,7 @@ jobs:
   - script: flake8 .
     displayName: "Run flake8"
 
-  - script: python -m pytest test/ --ignore=test/perf --junitxml=./TEST--TEST.xml -o junit_suite_name="$(Agent.JobName)"
+  - script: python -m pytest test/unit --junitxml=./TEST--TEST.xml -o junit_suite_name="$(Agent.JobName)"
```

Comment:
why this change? Explicitly ignoring means we'd catch issues in new tests that are in neither directory, while this now would ignore test/integration or whatever one might add

Reply:
Have changed, but we're getting to the point where these need to be regularised some more.